eosdis-nasa/earthdata-pub-api
# Edpub 861 add unknown daac workflow

# Description

Creates an unknown daac workflow.

## Spec

See Ticket: [EDPUB-861](https://bugs.earthdata.nasa.gov/browse/EDPUB-861)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Create a new user that is a member of no daac and a daac manager and staff
4. Create a new request using the unknown workflow and ensue the new user gets added to the request.
5. Confirm workflow runs to completion.

---

## Change Log

Creates an unknown daac workflow.

* [Add _____](link to commitid)
* [Fix _____](link to commitid)
